app/forms/aboutyou/CharitableDonationsFormProvider.scala

```diff
@@ -14,26 +14,20 @@
  * limitations under the License.
  */
 
-package forms
-
-import javax.inject.Inject
+package forms.aboutyou
 
 import forms.mappings.Mappings
+import models.aboutyou.CharitableDonations
 import play.api.data.Form
 import play.api.data.Forms.set
-import models.CharitableDonations
+
+import javax.inject.Inject
 
 class CharitableDonationsFormProvider @Inject() extends Mappings {
 
   def apply(isAgent: Boolean): Form[Set[CharitableDonations]] = {
-    val error: String =
```

Comment:
This needs to stay as we will get the error messages from Design team.Every form needs some checks